library/core/registry/src/main/java/org/quiltmc/qsl/registry/impl/sync/SynchronizedRegistry.java

```diff
@@ -23,7 +23,9 @@
 
 import net.minecraft.registry.Registry;
 import net.minecraft.util.Identifier;
+import org.quiltmc.qsl.registry.impl.sync.registry.RegistryFlag;
 
+// TODO - Move to registry package once qfapi is fixed
```

Comment:
wait, why aren't you moving because of QFAPI? breaking QFAPI is acceptable because i have to update it anyway in order for it to include the latest QSL update!

Reply:
Needed that to test if it works on prod with few mods

Reply:
Ah, that's fair enough; maybe i should investigate patching up a QFAPI build with the thingy fixed :p